packages/medusa/src/api/routes/store/product-categories/get-product-category.ts

```diff
@@ -38,7 +38,7 @@ import { defaultStoreScope } from "."
  *   - api_token: []
  *   - cookie_auth: []
  * tags:
- *   - Product Category
+ *   - Product Categorys
```

Comment:
Should be ProductCategories

Reply:
🤦 , fixed in 32e57f882c082c16e00fa96cd57e722d6001b100